sites/public/pages/applications/contact/address.tsx

```diff
@@ -118,6 +118,9 @@ const ApplicationAddress = () => {
     id: item.id,
     label: t(`t.${item.id}`),
     defaultChecked: application?.contactPreferences?.includes(item.id) || false,
+    disabled:
+      ((item.id === "phone" || item.id === "text") && noPhone && true) ||
```

Comment:
I don't think you need the `&& true`, it may also be a good idea to pull this out into a separate  helper function since if we ever allowed an application with no current address we would need to do something different with this